kibana/templates/clusterrole.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.managedServiceAccount }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ template "kibana.serviceAccount" . }}-cluster-role
+  labels:
+    app: "{{ template "kibana.fullname" . }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+rules:
```

Comment:
a specific rule for `podsecuritypolicies` resource is required similar to https://github.com/elastic/helm-charts/pull/496#pullrequestreview-431550555